app/config.py

```diff
@@ -134,8 +134,7 @@ class Config(object):
     AWS_SES_SMTP = os.getenv("AWS_SES_SMTP", "email-smtp.us-east-1.amazonaws.com")
     AWS_SES_ACCESS_KEY = os.getenv('AWS_SES_ACCESS_KEY')
     AWS_SES_SECRET_KEY = os.getenv('AWS_SES_SECRET_KEY')
-    AWS_PINPOINT_APP_ID = os.getenv('AWS_PINPOINT_APP_ID')
-    AWS_PINPOINT_KEYWORD = os.getenv('AWS_PINPOINT_KEYWORD')
+    AWS_PINPOINT_REGION = os.getenv('AWS_PINPOINT_REGION', "us-west-2")
```

Comment:
Will had this new env on k8s, but even if it's not defined (by mistake) the fallback value will work

Reply:
This is fine, but what do you think about certain env vars being required on app init? I've used this method in the past where we want to enforce specificity on the environment @ init as opposed to relying on implicit fallbacks.

Reply:
Sounds like a good idea for a future refactor! For now, config files usually have fallback but I agree we shouldn't rely on those to make things work.

I'll draft up a k8s PR

Reply:
@dorianjp Here is the k8s PR as promised https://github.com/cds-snc/notification-manifests/pull/150